pkts-core/src/main/java/io/pkts/frame/PcapRecordHeader.java

```diff
@@ -85,7 +92,7 @@ public long getTimeStampSeconds() {
         return PcapGlobalHeader.getUnsignedInt(0, this.body.getArray(), this.byteOrder);
     }
 
-    public long getTimeStampMicroSeconds() {
+    public long getTimeStampMicroOrNanoSeconds() {
```

Comment:
If you could leave the old method around as well just so that the current interface isn't broken. Mark the old one as deprecated.